Sources/MongoSwift/ChangeStreamDocument.swift

```diff
@@ -0,0 +1,76 @@
+/// An `UpdateDescription` containing fields that will be present in the change stream document
+/// for operations of type `update`.
+public struct UpdateDescription: Codable {
+   /// A document containing key:value pairs of names of the fields that were changed, and the new
+   /// value for those fields.
+   public let updatedFields: Document
+
+   /// An array of field names that were removed from the document.
+   public let removedFields: [String]
+}
+
+/// An enum representing the type of operation for this change event.
+public enum OperationType: String, Codable {
+   /// Specifies an operation of type `insert`.
+   /// SeeAlso: https://docs.mongodb.com/manual/reference/change-events/index.html#insert-event
```

Comment:
```suggestion
   /// - SeeAlso: https://docs.mongodb.com/manual/reference/change-events/index.html#insert-event
```
I think this might be needed for docs tool to pick up on them/format them correctly. same applies to the below `SeeAlso`s